### ISSUE TITLE
Recordings don't change back to published in moodle and BBB v2.6.8

### ISSUE THREAD

**Issue by more-zamani** [bigbluebutton/bigbluebutton#18038]
Hi
I installed the latest version of BBB plugin on Moodle v3.11.3, and I held a class. After publishing the recording of that class in the Recordings table, I clicked the eye button on the Toolbar column to unpublish the session. After some days, I clicked the eye button again to publish the recording, but in didn't work and I couldn't publish that session. 

I even use "mconf.github.io" to publish recording, but I did not succeed. Note that I have BBB v2.6.8

**Comment by hostbbb** [comment]
When you used the api mate, what xml response did you get when you tried to publish?   
you can look at nginx on bbb and get the exact api call moodle is sending to try and debug.

question.. can you unpublish and publish other old recordings? what about brand new recordings.

**Comment by amg-web** [comment]
@hostbbb 
can add here that meeting rename is not reflected in BBB from moodle
error in bbb-web.log:
```
2023-05-31T02:12:22.469Z DEBUG o.b.w.c.RecordingController - RecordingController#updateRecordings
	at org.bigbluebutton.api.MeetingService.updateRecordings(MeetingService.java:599)
	at org.bigbluebutton.api.MeetingService$updateRecordings$14.call(Unknown Source)
	at org.bigbluebutton.web.controllers.RecordingController.updateRecordingsHandler(RecordingController.groovy)
	at org.bigbluebutton.api.MeetingService.updateRecordings(MeetingService.java:599)
2023-05-31T02:12:22.558Z ERROR o.g.w.e.GrailsExceptionResolver - NullPointerException occurred when processing request: [GET] /bigbluebutton/api/updateRecordings
```
and request from moodle in nginx access log:
`13.239.125.105 - - [31/May/2023:04:09:38 +0000] "GET /bigbluebutton/api/updateRecordings?recordID=ccf5c313e269be27f9093b544a48c49dc95c1b75-1685503575591&meetingID=a662e21d80a7b06f837e03f1b2f6fb51d2524df3-308-26%5B0%5D&meetingName=FNSFMB411+-+Role+Play+-+Maria+REAA&published=false&state=published&startTime=1685503575591&endTime=1685505461920&playbacks%5Bpresentation%5D%5Btype%5D=presentation&playbacks%5Bpresentation%5D%5Burl%5D=https%3A%2F%2Fbbb.domain.com.au%2Fplayback%2Fpresentation%2F2.3%2Fccf5c313e269be27f9093b544a48c49dc95c1b75-1685503575591&playbacks%5Bpresentation%5D%5Blength%5D=30&playbacks%5Bpresentation%5D%5Bpreview%5D%5B0%5D%5Burl%5D=https%3A%2F%2Fbbb.domain.com.au%2Fpresentation%2Fccf5c313e269be27f9093b544a48c49dc95c1b75-1685503575591%2Fpresentation%2Fafd271de235da066577def9e6e017e72b9c70b72-1685503575597%2Fthumbnails%2Fthumb-1.png&playbacks%5Bpresentation%5D%5Bpreview%5D%5B0%5D%5Bwidth%5D=176&playbacks%5Bpresentation%5D%5Bpreview%5D%5B0%5D%5Bheight%5D=136&playbacks%5Bpresentation%5D%5Bpreview%5D%5B0%5D%5Balt%5D=Welcome+To&playbacks%5Bpresentation%5D%5Bpreview%5D%5B1%5D%5Burl%5D=https%3A%2F%2Fbbb.domain.com.au%2Fpresentation%2Fccf5c313e269be27f9093b544a48c49dc95c1b75-1685503575591%2Fpresentation%2Fafd271de235da066577def9e6e017e72b9c70b72-1685503575597%2Fthumbnails%2Fthumb-2.png&playbacks%5Bpresentation%5D%5Bpreview%5D%5B1%5D%5Bwidth%5D=176&playbacks%5Bpresentation%5D%5Bpreview%5D%5B1%5D%5Bheight%5D=136&playbacks%5Bpresentation%5D%5Bpreview%5D%5B1%5D%5Balt%5D=&playbacks%5Bpresentation%5D%5Bpreview%5D%5B2%5D%5Burl%5D=https%3A%2F%2Fbbb.domain.com.au%2Fpresentation%2Fccf5c313e269be27f9093b544a48c49dc95c1b75-1685503575591%2Fpresentation%2Fafd271de235da066577def9e6e017e72b9c70b72-1685503575597%2Fthumbnails%2Fthumb-3.png&playbacks%5Bpresentation%5D%5Bpreview%5D%5B2%5D%5Bwidth%5D=176&playbacks%5Bpresentation%5D%5Bpreview%5D%5B2%5D%5Bheight%5D=136&playbacks%5Bpresentation%5D%5Bpreview%5D%5B2%5D%5Balt%5D=&meta_bbb-origin-tag=moodle-mod_bigbluebuttonbn+%282022112800%29&meta_bbb-recording-tags=&meta_meetingName=FNSFMB411+-+Role+Play+-+Maria+REAA&meta_meetingId=a662e21d80a7b06f837e03f1b2f6fb51d2524df3-308-26%5B0%5D&meta_bbb-recording-description=%0A%0AAs+part+of+FNSFMB411+-+Assessment+Tasks+you+are+required+to+...&meta_bbb-origin=Moodle&meta_isBreakout=false&meta_bbb-origin-server-name=reaa.domain.com&meta_bbb-origin-server-common-name=&meta_bbb-origin-version=4.1+%28Build%3A+20221128%29&meta_bbb-recording-name=FNSFMB411+-+Role+Play+-+Maria+REAA&meta_bbb-context-label=Session+Bookings+%28FNS%29&meta_bbb-context-id=308&meta_bbb-context-name=Role+Play%2FTutorial+Session+Bookings+%28FNS%29&meta_bbb-context=Role+Play%2FTutorial+Session+Bookings+%28FNS%29&meta_bbb-meeting-size-hint=1686&checksum=66f5b818c46a2ef95f58d8afbfc0c4d12f3c0a85 HTTP/1.1" 200 87 "-" "MoodleBot/4.1 (+https://reaa.domain.com)"`
